utils/xmllite.d

```diff
@@ -508,9 +533,13 @@ void parseInto(Config)(XmlNode node, ref StringStream s)
 					{
 						while (true)
 						{
+							auto after_close_backup = s.backup;
```

Comment:
`after_close_backup` is used only in a `static if`-block. It can be defined in a `static if`-block as well.